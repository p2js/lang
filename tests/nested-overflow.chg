extern "intrinsic" fn bitcast<T, U>(v: T): U;

fn main() {
  let x = bitcast(0);
  x = bitcast(1);
}
