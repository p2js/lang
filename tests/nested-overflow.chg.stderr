type error: type annotations needed

  let x = bitcast(0);
          ^^^^^^^^^^
type error: type annotations needed

  x = bitcast(1);
      ^^^^^^^^^^
